Fix/vanta docs (#6323)
getsentry/sentry-docs: src/docs/product/integrations/compliance/vanta/index.mdx

---
title: Vanta
sidebar_order: 1
redirect_from:
  - /product/integrations/vanta/

> description: "Learn about Sentry's Vanta integration and how it to helps automate security compliance. ."

Reviewer: ```suggestion
description: "Learn about Sentry's Vanta integration and how it helps automate security compliance."
```

---

> The Vanta integration for Sentry grants Vanta read-only access to help verify that Sentry organization members are associated with employees in your organization and helps automate security compliance.
> 
> 
> This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).
> 

Reviewer: ```suggestion
Sentry's Vanta integration grants Vanta read-only access to verify that Sentry organization members are associated with employees in your organization, which automates security compliance. This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).

```

## Install and Configure

<Note>

Sentry owner, manager, or admin permissions are required to install this integration.

</Note>

> 
> 1. Navigate to **Settings > Integrations > Vanta**
> 
> 2. Follow the full [Vanta installation instructions](https://help.vanta.com/hc/en-us/articles/13045851267092).

Reviewer: ```suggestion

To set up the Vanta integration:

1. Navigate to **Settings > Integrations > Vanta**

2. Follow the full [Vanta installation instructions](https://help.vanta.com/hc/en-us/articles/13045851267092).
```
